gpt4all-chat/qml/Theme.qml

```diff
@@ -120,572 +120,572 @@ QtObject {
     property color accentColor: {
         switch (MySettings.chatTheme) {
             case "LegacyDark":
-                return blue200;
+                return blue200
```

Comment:
I don't understand how this fixes the indentation. Can you explain? Are you replacing spaces with tabs??